tools/emscripten.py

```diff
@@ -61,7 +61,7 @@ def compute_minimal_runtime_initializer_and_exports(post, exports, receiving):
   exports = [asmjs_mangle(x) for x in exports if x != building.WASM_CALL_CTORS]
 
   # Decide whether we should generate the global dynCalls dictionary for the dynCall() function?
-  if settings.DYNCALLS and '$dynCall' in settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE and len([x for x in exports if x.startswith('dynCall_')]) > 0:
+  if settings.DYNCALLS or not settings.WASM_BIGINT or '$dynCall' in settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE:
```

Comment:
Is this change suppose to be in here?

Reply:
Yeah, without this the simple test doesn't compile due to closure warning about `dynCalls` symbol being missing.

Reply:
Actually I found a better way to express this dependency precisely in library.js where it belongs.